docs/bxcodec.html

```diff
@@ -29,25 +29,25 @@ <h1 id="bxcodec-iman-tumorang"><span class="citation" data-cites="bxcodec">@bxco
 <a href="https://github.com/bxcodec">github.com/bxcodec</a><br />
 <a href="https://maintaine.rs/bxcodec">maintaine.rs/bxcodec</a></p>
 </blockquote>
-<p>I’m <a href="https://imantumorang.com/">Iman Tumorang</a>, currently working as a full-time engineer at <a href="https://veriff.com">Veriff</a>, one of the biggest ID verification solution startups in Estonia. Prior to this, I also worked in a couple of industries, such as payment, media, and CRM. Throughout my professional journey, I’ve been following the open-source community. Even though I haven’t made considerable contributions to any popular projects, I can proudly say I’ve been maintaining a couple of my Golang libraries, which a few people fortunately use.</p>
-<p>I noticed a correlation between my professional career and open-source contributions. All my Golang libraries are inspired by the problems I encounter in my work. Sometimes, I need to create a small library to speed up development time. If I see that it can be helpful in the public, I will make it open-source; if not, it will remain an internal source within the company.</p>
+<p>I’m <a href="https://imantumorang.com/">Iman Tumorang</a>, currently working as a full-time engineer at <a href="https://veriff.com">Veriff</a>, one of the biggest ID verification solution startups in Estonia. Prior to this, I also worked in a couple of industries, such as payment, media, and CRM. Throughout my professional journey, I’ve been following the Open Sourcecommunity. Even though I haven’t made considerable contributions to any popular projects, I can proudly say I’ve been maintaining a couple of my Golang libraries, which a few people fortunately use.</p>
```

Comment:
Looks like the word got combined together resulting in `Sourcecommunity` which is an issue.

Reply:
This with all changed files